sktime_dl/deeplearning/base/estimators/_classifier.py

```diff
@@ -93,16 +93,25 @@ def check_and_clean_data(self, X, y=None, input_checks=True):
             else:
                 validate_X_y(X, y)
 
+        # want data in form: [instances = n][timepoints = m][dimensions = d]
         if isinstance(X, pd.DataFrame):
-            if X.shape[1] > 1 or not isinstance(X.iloc[0, 0], pd.Series):
-                raise TypeError(
-                    "Input should either be a 2d numpy array, or a pandas dataframe with a single column of Series objects (networks cannot yet handle multivariate problems")
+            if X.shape[1] > 1:
+                # we have multiple columns
+                # if cells contain series, this is a multidimensional problem
+                # else if cells contain single values, this is a univariate problem with values long columns
+                #      this situation can happen with e.g. forecasting-reduced-to-regression strategies
+                if isinstance(X.iloc[0, 0], pd.Series):
```

Comment:
I'd make this check also into a function, that'll make it easier later one if we change the data container, i.e. something like
```python
def is_nested_dataframe(X):
    return isinstance(X.iloc[0, 0], pd.Series)
```